Updated BUILDING.md to prevent some confusion

ThatWeirdAndrew opened the pull request:
After I updated the DistroLauncher-Appx.vcxproj file. I found that there was already a pull request to fix the problem. I don't know how to leave it out but you might be able to remove that commit.

crramirez commented:
Hello @thatweirdandrew

Please put me in the context. What problem are you trying to solve? Which is the pull request and commit you are talking about? 

Thanks

ThatWeirdAndrew commented:
I was fixing small errors that could have led to some confusion. Before, the build requirements left out the Windows 10 SDK 17763 which is required so I added that. Also, the original build instructions just told you to generate a test certificate but it left out the part to update the certificate in the project files so I added that. Commit 8f6b437 updates the path for the finished app files so it is the same as the BUILDING.md file. What I didn't notice was that it was already fixed in PR #522.

crramirez commented:
Ok, so when we merge, it probably will generate conflicts. No problem. Thank you
